source/Ship.cpp

```diff
@@ -3087,6 +3090,11 @@ int Ship::Crew() const
 }
 
 
+int Ship::CrewValue() const
+{
+	return max(Crew(), RequiredCrew()) + attributes.Get("crew equivalent");
+}
```

Comment:
Functions should be defined in the .cpp file in the same order that they're listed in the .h file. In the header you've listed CrewValue() below AddCrew(), but in the .cpp file you've defined CrewValue() below Crew().

I'd change it so that in both locations, CrewValue() is located below RequiredCrew().

Also make sure that there are three lines of whitespace above and below the function.

Reply:
well because you _really_ wanted it I've placed CrewValue() above AddCrew() in both cases (ik you prob forgot about this one, maybe it's git bugging a bit too)